TTS/utils/arguments.py

```diff
@@ -192,6 +179,12 @@ def process_args(args, model_type):
         if args.restore_path:
             new_fields["restore_path"] = args.restore_path
         new_fields["github_branch"] = get_git_branch()
+        # if model characters are not set in the config file
+        # save the default set to the config file for future
+        # compatibility.
+        if model_class == 'TTS' and not 'characters' in c.keys():
```

Comment:
This should work on dicts:

```suggestion
        if model_class == 'TTS' and not 'characters' in c:
```